docs/concepts/EvictionAndDisruptionBudget/README.md

```diff
@@ -0,0 +1,47 @@
+# Eviction & Placement Disruption Budget
+
+This document explains the concept of `Eviction` and `Placement Disruption Budget` in the context of the fleet.
+
+## Overview
+
+`Eviction` pertains to the act of removing resources from a target cluster propagated by a resource placement object from the hub cluster.
+
+The `Placement Disruption Budget` object protects against voluntary disruption, and in the case of the fleet, the only allowed voluntary disruption as of now is eviction.
+
+## ClusterResourcePlacementEviction
+
+An eviction object is used to remove resources from a member cluster once the resources have already been propagated from the hub cluster.
+
+Once the resources are successfully removed or if eviction cannot be executed, the eviction object won't be reconciled again.
+
+To successfully evict resources from a cluster, the user needs to specify:
+
+- The name of the `ClusterResourcePlacement` object which propagated resources to the target cluster
+- The name of the target cluster from which we need to evict resources
+
+When specifying the `ClusterResourcePlacement` object in the eviction's spec, the user needs to consider the following cases:
+
+- For `PickFixed` CRP, eviction is not allowed because if users wanted to remove resources from a cluster, they could choose to remove the cluster name from the `ClusterResourcePlacement` spec or pick a different cluster.
```

Comment:
A nit: for simplicity reasons maybe we could say that eviction only works with CRPs of the `PickFixed` and `PickAll` placement types; for CRPs of the `PickFixed` type, it is recommended that one directly edit the list of target clusters on the CRP object.